data/remnant/remnant jobs.txt

```diff
@@ -223,6 +224,115 @@ mission "Remnant: Bounty 5"
 
 
 
+mission "Remnant: Logistics Civil Engineering"
+	name "Civil delivery to <planet>"
+	description `Engineers on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant civil" 30 80
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
+		not "flagship planet: Ssil Vida"
+		has "Remnant: Logistics 1: done"
+		random < 51
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A project engineer inspects the <cargo> and signs off on your payment of <payment>.`
+		payment 10000 15
+
+
+
+mission "Remnant: Logistics Industrial Delivery"
+	name "Industrial delivery to <planet>"
+	description `Project teams on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant industrial" 30 300
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
+		not "flagship planet: Ssil Vida"
+		has "Remnant: Logistics 1: done"
+		has "Remnant: Logistics Civil Engineering: done" >= 9
+		random < 76
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A project technician inspects the <cargo> and approves your payment of <payment>.`
+		payment 20000 25
+
+
+
+mission "Remnant: Logistics Outfitter Delivery"
+	name "Outfitter component delivery to <planet>"
```

Comment:
```suggestion
	name "Component delivery to <planet>"
```
The existing name might be too long and run off the visible space in the job title "tab" when viewed in-game.